ceph-windows-image-build/build/setup.ps1

```diff
@@ -125,7 +130,12 @@ function Install-Tool {
             Start-FileDownload -URL $URL -Destination $installerPath
         }
         Write-Output "Installing ${installerPath}"
-        $p = Start-Process -FilePath $installerPath -ArgumentList $Params -NoNewWindow -PassThru -Wait
+        if($MSI) {
```

Comment:
I believe he can simply check if the `${installerPath}` ends with `.msi` and don't need an extra `$MSI` function parameter.